Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbarSettings.cpp

```diff
@@ -1,9 +1,11 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 #include "SpatialGDKEditorToolbarSettings.h"
 
-USpatialGDKEditorToolbarSettings::USpatialGDKEditorToolbarSettings(const FObjectInitializer& ObjectInitializer) : SpatialOSLaunchConfig(TEXT("default_launch.json")), bStopSpatialOnExit(false), Super(ObjectInitializer)
+USpatialGDKEditorToolbarSettings::USpatialGDKEditorToolbarSettings(const FObjectInitializer& ObjectInitializer) : SpatialOSLaunchConfig(TEXT("default_launch.json")), bStopSpatialOnExit(false),
```

Comment:
Split each entry in the initializer list on a newline:

Constuctor()
    : Var1
   , Var2
...

Also call Super first in the list